store/goredisstore/goredisstore_test.go

```diff
@@ -0,0 +1,94 @@
+package goredisstore_test
+
+import (
+	"log"
+	"testing"
+	"time"
+
+	"github.com/go-redis/redis"
+	"github.com/throttled/throttled"
+	"github.com/throttled/throttled/store/goredisstore"
+	"github.com/throttled/throttled/store/storetest"
+)
+
+const (
+	redisTestDB     = 1
+	redisTestPrefix = "throttled-go-redis:"
+)
+
+// Demonstrates that how to initialize a RateLimiter with redis
+// using go-redis libaray.
```

Comment:
Thanks for the example! Typo here: "library".

Reply:
Oops 😅. Fixed! 😸